http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -563,4 +564,16 @@ public HttpServerConfig setHttp2StreamIdleTimeout(Duration http2StreamIdleTimeou
         this.http2StreamIdleTimeout = http2StreamIdleTimeout;
         return this;
     }
+
+    public boolean isLogCompressionEnabled()
```

Comment:
Move these to be next to the other log-related methods.

BTW, separately, we should rename `queueSize` and the methods to be `logQueueSize`.